phantazm-commons/src/main/java/com/github/phantazmnetwork/commons/vector/VectorConfigProcessors.java

```diff
@@ -0,0 +1,84 @@
+package com.github.phantazmnetwork.commons.vector;
+
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.ConfigPrimitive;
+import com.github.steanky.ethylene.core.collection.ConfigNode;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import org.jetbrains.annotations.NotNull;
+
+public final class VectorConfigProcessors {
+    private static final String X_COMPONENT_STRING = "x";
+    private static final String Y_COMPONENT_STRING = "y";
+    private static final String Z_COMPONENT_STRING = "z";
```

Comment:
I don't particularly believe that we should be doing this, it gets a bit unruly for complex objects

Reply:
that's fair, but they're used in two places and as annoying as fields like this are, i dislike duplicate strings even more
any ideas for alternatives?